ui/src/app/components/gke-app-configuration-panel.tsx

```diff
@@ -136,18 +126,38 @@ export const GKEAppConfigurationPanel = ({
     panelContent,
     [
       GKEAppPanelContent.CREATE,
-      () => (
-        <CreateGKEAppPanel
-          {...{
-            ...props,
-            appType,
-            app,
-            disk,
-            onClickDeleteUnattachedPersistentDisk,
-            onClose,
-          }}
-        />
-      ),
+      () =>
+        switchCase(
+          appType,
+          [
+            AppType.CROMWELL,
+            () => (
+              <CreateCromwell
```

Comment:
This naming convention seems more sensible.

panel content = create
type = Cromwell

component -> CreateCromwell